cloudfront/wellcomelibrary.org/terraform/cf_behaviours.tf

```diff
@@ -103,7 +103,6 @@ locals {
 
   prod_behaviours = concat(
     local.api_behaviours,
-    local.items_behaviours
```

Comment:
Can you now delete the associated local?

Reply:
I'll let them hang around for a bit in case I need to undo this action and don't want to fiddle with git reverts.